Cargo.toml

```diff
@@ -29,7 +29,7 @@ unsigned-varint = { version = "0.7.1", features = ["futures", "std"] }
 [dev-dependencies]
 async-std = { version = "1.10.0", features = ["attributes"] }
 env_logger = "0.9.0"
-libipld = { version = "0.14.0", default-features = false, features = ["dag-cbor"] }
+libipld = { version = "0.15.0", default-features = false, features = ["dag-cbor"] }
```

Comment:
This will require many other crates to upgrade as well. Are there reasons to make this urgent? Otherwise I’d have postponed this to the next libp2p upgrade.

Reply:
I am using libipld 0.15 which applies this [fix](https://github.com/ipld/libipld/commit/524cffa222d0e68868cb1b5db1ab30dbc386bca9) and because of using a newer version, libp2p-bitswap would give a compiler error due to the use of `StoreParams` from a newer version (which might fall back to a previous comment made in https://github.com/ipfs-rust/libp2p-bitswap/pull/39). If you wish to postpone, I can use a fork with this upgrade for the time being until it is updated here

Reply:
Right, those compilation errors are what then forces the upgrade dance on all my repos … we can merge this now but I’d like to hold back with releasing until libp2p 0.51 is out.

Reply:
Waiting until libp2p 0.51 is fine with me.